docs/core-concepts/security/security-model.md

```diff
@@ -0,0 +1,200 @@
+- Each chunk can be paired with an opening proof which can be used to verify that the chunk was properly derived from a blob corresponding to a particular commitment.
+
+To achieve these properties, the module provides the following primitives: 
+
+- `EncodeAndProve`. Extends a blob of data into a set of encoded chunks. Also produces opening proofs and blob commitments.
+- `Verify`. Verifies a chunk against a blob commitment using an opening proof.
+- `Decode`. Reconstructs the original blob given a sufficiently sized collection of encoded chunks.
+
+EigenDA implements the encoding module using Reed Solomon encoding, together with KZG polynomial commitments and opening proofs. More details about the encoding and proving module can be found in the [code spec](https://github.com/Layr-Labs/eigenda/blob/master/docs/spec/src/protocol/architecture/encoding.md).
+
+# Security Models
+
+In EigenDA, there are three different kinds of quorums where different assets (restaked-ETH, EIGEN and customized tokens of roll-ups) are delegated to the operators.  Different quorums provide different security guarantees. All three kinds of quorums must simultaneously fail for a safety attack to be successfully executed, providing multi-layered security assurance.
+
+The three security models in EigenDA, along with their corresponding quorums, are outlined below:
+
+- BFT security: ETH, EIGEN and Custom Quorum
+- Cryptoeconomic security: EIGEN Quorum
+- Token Toxicity: Custom Quorum
+
+We begin by giving an overview of each security model and how it contributes to EigenDA’s overall resilience: BFT security ensures both safety and liveness of the system as long as the share of stake or voting power held by malicious validators stays below a certain threshold. Cryptoeconomic security goes a step further—if an attacker misbehaves, they not only need to control a significant amount of stake, but they also risk losing it through slashing. This makes attacks financially unappealing. Token toxicity adds another layer of incentive alignment. When validators misbehave, the native token may drop in value, leading to losses for token holders who delegated their stake to those validators. This dynamic encourages stakeholders to carefully choose trustworthy operators.
+
+In the rest of this page, we provide a detailed analysis of how the three kinds of security are satisfied.
+
+## BFT Security Model
+
+The BFT security model ensures system safety and liveness as long as the stake delegated to malicious validators remains below a predefined threshold.
+
+### Security Parameters
+
+The allocation of blob data among the EigenDA validators is governed by Chunk Assignment logic which takes as input a set of `BlobParameters` which are linked to the blob `Version` by a mapping in the `EigenDAServiceManager` contract. The `BlobParameters` consist of:
+
+- `NumChunks` - The number of encoded chunks that will be generated for each blob (must be a power of 2).
```

Comment:
I think you should either explain why it must be a power of 2 if it is relevant to security, or not mention it here. I believe it is a KZG detail irrelevant to this doc?